core/base/src/TROOT.cxx

```diff
@@ -2737,6 +2739,35 @@ void TROOT::SetMacroPath(const char *newpath)
       macroPath = newpath;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// If `web` is not empty it defines where web graphics should be rendered
+///
+/// - "cef" web graphics is rendered via the Chromium Embedded Framework
+/// - "qt5" web graphics is rendered via Qt5
+/// - "browser" or "" web graphics is rendered in the default web browser
+/// - "off" turn off the web display and come back to normal mode.
+///
+/// It also turn the normal graphics to Batch to avoid the loading of local
+/// graphics libraries.
+
+void TROOT::SetWebDisplay(const char *webdisplay)
+{
+   fWebDisplay = webdisplay;
+   fWebDisplay.ToLower();
+   if (fWebDisplay.Contains("cef")     || fWebDisplay.Contains("qt5") ||
+       fWebDisplay.Contains("browser") || fWebDisplay.Length() ==0) {
+      fIsWebDisplay = kTRUE;
+      if (gROOT->IsBatch()) fIsWebDisplayBatch = kTRUE;
```

Comment:
@couet 
I propose to have independent method - TROOT::SetWebDisplayBatch to change it independently.
One should be able to change it after starting ROOT.

Reply:
@linev 
I can interpret only "off" to turn off the web display and comme back to normal graphics. All the other strings will be stored "as they are" without any check. To avoid to have to many additional methods I can also look at the option "batch". So if one to come back to normal graphics mode interactively he will call it with "off" and if he wants to come back to normal graphics in batch he will call it with "off batch"

Reply:
@couet 
Yes, it could be good idea. But still one need direct method to change fIsWebDisplayBatch value.

Reply:
Since this 'WebDisplayBatch' is temporary, i would really prefer it was clearly marked as such, i.e. a setter probably should be private in TROOT itself and be accessible only via a ROOT::Experimental free standing function ....